src/main/scala/edu/ie3/simona/model/participant/BmModel.scala

```diff
@@ -37,9 +47,11 @@ final case class BmModel(
     override val cosPhiRated: Double,
     override val qControl: QControl,
     private val isCostControlled: Boolean,
-    private val opex: Money,
+    private val capex: Money,
+    private val opex: EnergyPrice,
     private val feedInTariff: EnergyPrice,
     private val loadGradient: Double,
+    private val isMarketReaction: Boolean,
```

Comment:
Is this used at all? If not, I'd remove it. One can always add it back in, if it's needed.